pkg/network/ovn_kubernetes.go

```diff
@@ -113,6 +113,19 @@ func fillOVNKubernetesDefaults(conf, previous *operv1.NetworkSpec, hostMTU int)
 		}
 		sc.MTU = &mtu
 	}
+
```

Comment:
This isn't necessary - there's no kube-proxy-config for ovn-kubernetes.

Reply:
Done